src/components/ProductListItem/index.tsx

```diff
@@ -0,0 +1,37 @@
+import BaseComponent from "@/components/BaseComponent";
+import { Component, Prop } from "vue-property-decorator";
+import Headline from "@/components/Headline";
+import RichText from "@/components/RichText";
+
+import "./style.css";
+import { ProductListItemProps } from "@/types/components";
+
+@Component({
+  name: "FSXAProductListItem",
```

Comment:
Please change the "displayName"  to `ProductListItem`

Reply:
done